src/app/Fake.Core.Trace/TraceListener.fs

```diff
@@ -129,6 +129,7 @@ type TagStatus =
     | Success
     | Warning
     | Failed
+    | FailedWithMessage of message:string
```

Comment:
maybe we could include the exception object.
Note that I'm a bit worried about this change... This is because the TagStatus is used on other places and I feel like adding a new state here needs more updates...